app/src/main/java/org/mozilla/fenix/settings/logins/view/LoginDetailView.kt

```diff
@@ -4,62 +4,19 @@
 
 package org.mozilla.fenix.settings.logins.view
 
-import android.text.InputType
 import android.view.ViewGroup
-import androidx.core.content.res.ResourcesCompat
 import kotlinx.android.extensions.LayoutContainer
 import kotlinx.android.synthetic.main.fragment_login_detail.*
-import kotlinx.android.synthetic.main.fragment_login_detail.view.*
-import org.mozilla.fenix.R
-import org.mozilla.fenix.components.metrics.Event
-import org.mozilla.fenix.ext.components
 import org.mozilla.fenix.settings.logins.LoginsListState
 
 /**
  * View that contains and configures the Login Details
  */
 @Suppress("ForbiddenComment")
```

Comment:
nit -  we may be able to remove this suppress now (I think it was about the TODO)